server/server.js

```diff
@@ -2,11 +2,12 @@ var express = require('express');
 var app = express();
 var port = 8000;
 var clearDatabase = (process.env.NODE_ENV !== 'production' && process.env.CLEAR_DATABASE === 'true')
+var server = require('http').createServer(app);
```

Comment:
hey - do you need to use the http library? check out socket.io's docs: http://socket.io/docs/#using-with-express-3/4